src/Pages/Patients/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Route, Switch } from "react-router-dom"
+
+import Edit from "./Edit"
+
+
+Patients.propTypes = {
+  match: PropTypes.object
+}
+
+function Patients({ match }) {
+  return (
+    <Switch>
+      <Route path={`${match.url}/create-new`} component={Edit} />
+      <Route path={`${match.url}/edit/:id`} component={Edit} />
+      
+      
```

Comment:
You left 2 lines empty here. Prettier should have caught this and delete both of them. Make sure Prettier is up and running on your VS code